CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+Contributing
+============
+
+Thanks for helping out!
+
+## Checks, Tests
+
+Run `npm run builder:check`
+
+## Documentation
+
+To generate the usage snippet in the `README.md`, run:
+
+```sh
+$ builder help builder-react-component
```

Comment:
The problem is splitting up README and inserting it in, then committing it. Oh, and needs to run from `PROJECT` not archetype, although I can probably address that.... (but the archetype doesn't actually have `builder` as a dep...)